tests/test_chronos_tools.py

```diff
@@ -912,7 +917,7 @@ def test_format_chronos_job_dict_invalid_param(self):
             branch_dict={},
         )
         with raises(chronos_tools.InvalidChronosConfigError) as exc:
-            invalid_config.format_chronos_job_dict('', [])
+            invalid_config.format_chronos_job_dict('', [], {})
```

Comment:
yes please
http://paasta.readthedocs.org/en/latest/style_guide.html#positional-args-versus-keyword-args

please make them all kwargs while you are at it. I understand that changing the other args is "technically" not related to this change, but it isn't that big of of a deal to make them consistent while you are in here.